pkgs/tools/admin/otc-auth/default.nix

```
@@ -0,0 +1,31 @@
+{ pkgs, lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "otc-auth";
+  version = "2.0.3";
+
+  src = pkgs.fetchFromGitHub {
```

```suggestion
{ lib, buildGoModule, fetchFromGitHub }:

buildGoModule rec {
  pname = "otc-auth";
  version = "2.0.3";

  src = fetchFromGitHub {
```

For packages, you need to explicitly add each dependencies in the imports without referencing pkgs.